### DISCUSSION TITLE
Filament project show 404 not found error on shared host server

### DISCUSSION THREAD

**Discussion by Quids-Byte** [filamentphp/filament#9846] [Help]
Hi there,
My project was functioning correctly on my local environment, but when I deployed it on the Cpanel shared host, the /admin page was giving a 404 not found error.

**Comment by leandrocfe** [comment]
I don't use cpanel + laravel but [this article](https://laravel-school.com/posts/how-to-deploy-laravel-project-on-cpanel-2023/) can help you I think.
